themes/openy_themes/openy_rose/scss/modules/_global-search.scss

```diff
@@ -46,4 +46,13 @@
       font-weight: bold;
     }
   }
+
+  .form-item {
+    margin: 0;
+  }
+}
+
+.paragraph--type--search-api {
+  padding: 20px 15px !important;
```

Comment:
same here, please
Seems like the rule could be at least merged into single class if necessary

Reply:
Fixed